source/features/dim-bots.tsx

```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-const commitSelector = [
-	/* Commits */
-	'.commit-author[href$="%5Bbot%5D"]',
-	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
-].join(',');
+const commitSelectors = [
+	'actions-user',
+	'bors',
+	'imgbot',
```

Comment:
😅 Oops, I never read the actual list here and there are some error here, for example @imgbot should be @imgbotapp

Reply:
Fixed in https://github.com/refined-github/refined-github/commit/79ef6d8f680ab39c17bb6ec4458b2228769e5fe8